benchmarks/setup.rb

```diff
@@ -39,7 +39,7 @@ def tags
 
 def users_with_combined_tasks
   @users_with_combined_tasks ||= rom.relation(:users).as(:user_with_combined_tasks)
-    .combine(rom.relation(:tasks).for_users)
+                                 .combine(rom.relation(:tasks).for_users)
```

Comment:
We decided not to do that. Not sure why rubocop is still configured to report that as an offense